doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -451,3 +452,340 @@ away from that end. Also a clockwise moment of 5 KN-m is applied at the overhang
               48        12         12          16            2     
     ───────────────────────────────────────────────────────────────
                                   E⋅I  
+
+Example 7
+---------
+
+There is a beam of length l, fixed at both ends. A concentrated point load
+of magnitude F is applied in downward direction at mid-point of the
+beam.
+
+::
+
+  \\\\|                  F                  |\\\\
+  \\\\|                  |                  |\\\\
+  \\\\|                  V                  |\\\\
+  \\\\|_____________________________________|\\\\
+  \\\\|_____________________________________|\\\\
+  \\\\|                  :                  |\\\\
+  \\\\|                  :                  |\\\\
+  \\\\|------------------|------------------|\\\\
+               l/2                l/2            
+
+.. note::
+
+    Using the sign convention of upward forces and clockwise moment
+    being positive.
+
+.. code:: pycon
+
+    >>> from sympy.physics.continuum_mechanics.beam import Beam
+    >>> from sympy import symbols
+    >>> E, I, F = symbols('E I F')
+    >>> l = symbols('l', positive=True)
```

Comment:
Should we always have `real=True` also?

Reply:
E and I should always be positive too.

Reply:
I don't think there is a need for `real=True`. `positive=True` is used to simplify terms like `SingularityFunction(L, 0, 1)` to simply `L`.
Have a look if you what happens we dont set `positive=True` : https://pastebin.com/jrXWg2V8

Reply:
Making `E` and `I` positive should have no affect on solutions as they never are used inside `SingularityFunction`.

Reply:
Ok